src/backend/columnar/sql/downgrades/columnar--11.0-2--11.0-1.sql

```diff
@@ -0,0 +1,23 @@
+-- columnar--11.0-2--11.0-1.sql
+
+-- delete pg_depend records inserted in columnar--11.0-1--11.0-2.sql
```

Comment:
I would say that we should have a no-op downgrade script, because this downgrade will create a bug. If we had a no-op downgrade, we wouldn't re-introduce the bug. (Not sure what happens if there are multiple entries in pg_depend). What do you think about this?

Reply:
yeah, I guess Nils was also inclined to not doing anything in downgrade, so I should do this change 👍

> (Not sure what happens if there are multiple entries in pg_depend)

I tested that yesterday and there is no harm doing that.

Reply:
Actually I'm not sure if we should really remove downgrade steps. Isn't it mostly the case we rollback the changes during downgrade, even if it fixes a bug ?

Reply:
I guess as long as we are not changing functions etc it should be fine to have no-op downgrades for bug fixes like this one. 

What would be the downside of having no-op downgrade for this fix?

Reply:
I don't see any downside in doing that, it was only about what we've done so far in downgrade scripts.

Reply:
> Not sure what happens if there are multiple entries in pg_depend

> I tested that yesterday and there is no harm doing that.

Although there is seemingly no harm I would advise to make the insert such that it will select only the dependencies that are not already in pg_depend. Unless you find a path in postgres how a dependency can be added twice I am inclined to treat the entries in pg_depend as unique at this moment.

If in a future pg decides to add a unique constraint / primary key on the table we would have troubles with upgrades again.

Preventing this now will go a long way! You can probably use an `INSERT ... ON CONFLICT (DO NOTHING)` with the key on all the attributes.